def __init__(self, host='localhost', port=3306, user='cloud', passwd='cloud', db='cloud'):
    self.host = host
    self.port = port
    self.user = user
    self.passwd = passwd
    self.database = db
    try:
        self.db = pymysql.Connect(host=host, port=port, user=user, passwd=passwd, db=db)
    except:
        raise cloudstackException.InvalidParameterException(sys.exc_info())
